src/main/java/frc/robot/Robot.java

```diff
@@ -1,17 +1,39 @@
 package frc.robot;
 
 import com.chopshop166.chopshoplib.commands.CommandRobot;
+import com.chopshop166.chopshoplib.controls.ButtonXboxController;
+
+import frc.robot.maps.RobotMap;
+import frc.robot.subsystems.Climber;
 
 public class Robot extends CommandRobot {
 
+  private final ButtonXboxController controller = new ButtonXboxController(0);
+
+  private final RobotMap map = getMapForName("Gaston", RobotMap.class, "frc.robot.maps");
+
+  private final Climber leftClimber = new Climber(map.getLeftTelescopeMap());
+  private final Climber rightClimber = new Climber(map.getRightTelescopeMap());
+
   @Override
   public void robotInit() {
-    // Nothing to initialize
+    super.robotInit();
   }
 
   @Override
   public void configureButtonBindings() {
-    // TODO Auto-generated method stub
+
+    // Button bindings for regular climbing
+    controller.a().whileHeld(parallel("Extend", leftClimber.extend(), rightClimber.extend()));
+    controller.b().whileHeld(parallel("Retract", leftClimber.retract(), rightClimber.retract()));
+
+    // Button bindings for ignoring limit switches
+    controller.x()
```

Comment:
These should probably be on the d-pad where they're harder to press accidentally